src/gazebo_mavlink_interface.cpp

```diff
@@ -528,28 +532,22 @@ void GazeboMavlinkInterface::OnUpdate(const common::UpdateInfo& /*_info*/) {
     // turning_velocities_msg->header.stamp.sec = current_time.sec;
     // turning_velocities_msg->header.stamp.nsec = current_time.nsec;
 
-    // gzerr << turning_velocities_msg.motor_speed(0) << "\n";
+    
     motor_velocity_reference_pub_->Publish(turning_velocities_msg);
   }
 
   last_time_ = current_time;
 
-  //send gps
   math::Pose T_W_I = model_->GetWorldPose(); //TODO(burrimi): Check tf.
   math::Vector3 pos_W_I = T_W_I.pos;  // Use the models' world position for GPS and pressure alt.
 
-  math::Vector3 velocity_current_W = model_->GetWorldLinearVel();  // Use the models' world position for GPS velocity.
-
-  math::Vector3 velocity_current_W_xy = velocity_current_W;
-  velocity_current_W_xy.z = 0;
-
-  // TODO: Remove GPS message from IMU plugin. Added gazebo GPS plugin. This is temp here.
   // reproject local position to gps coordinates
```

Comment:
this is repeating the step done on the `reproject` method of `gazebo_gps_plugin` and should be removed. The `lat_rad` and `lon_rad` should be threaten as global variables of `gazebo_mavlink_interface` and updated each time a GPS callback is done, not here.